comp/core/workloadmeta/types.go

```diff
@@ -406,6 +466,20 @@ type Container struct {
 	Owner           *EntityID
 	SecurityContext *ContainerSecurityContext
 	Resources       ContainerResources
+
+	// ECS specific fields
```

Comment:
As discussed, I think it would be better not to include fields that are relevant only on specific environments. 

For users, it might be confusing why the field is empty in certain environments (for example why Health is not populated in docker environment).

For other developers, when getting a container entity, it would be tempting to do `c.Health` or `c.Volumes`, while this will only return data in specific environments.

A suggestion of @wdhif is to create a new entity `ECSContainer` and embed the existing `Container` entity within it. Something like this:

```
type ECSContainer struct {
Container
Limits    map[string]uint64
// other fields
}
```